src/styles.ts

```diff
@@ -105,7 +105,7 @@ export const adStyles = css`
                 line-height: 24px;
 
                 span {
-                    margin-right: 24px;
+                    margin-right: 30px;
```

Comment:
`remSpace`?

Reply:
The whole advert placeholder is set in pixels and I think the native layers only request ads at a certain pixel. We also have a ticket to use the icons in src instead of fonts (had a quick look into that and it requires a bit of refactoring).

Will merge this in now as it's just changing the alignment slightly.